webapp/templates/partials/scoreboard.html.twig

```diff
@@ -36,7 +36,9 @@
                     {{ now | printremainingminutes(current_contest.endtime) }}
                 {% else %}
                     {% if current_contest.freezeData.started %}
+                        {% set now = 'now'|date('U') %}
                         started:
+                        {{ current_contest.starttime | printelapsedminutes(now) }} at
```

Comment:
I'd prefer to keep it simple and use `printtimediff` here. That also renders something like `221 minutes` more readably as `3:41`.

Reply:
I think the point was to show exactly 221 minutes since you don't need to do further math to relate to the numbers in the scoreboard.